auction-server/src/auction/service/verification.rs

```diff
@@ -2216,7 +2228,7 @@ mod tests {
                 token_program_searcher: spl_token::id(),
                 fee_token,
                 referral_fee_bps,
-                user_mint_user_balance: 0,
```

Comment:
how were all these passing previously?

Reply:
the `user_mint_user_balance` wasn't used for anything